grommet/grommet
# Toast prevent interaction with elements after disappearing fix

#### What does this PR do?
Fix issue #1572 

#### What testing has been done on this PR?
None

#### How should this be manually tested?
Create a Toast component hiding Buttons or other interactable elements. When the Toast completely disappear, check to see if you can interact with the Button.

#### What are the relevant issues?
#1572 

#### Do the grommet docs need to be updated?
No

#### Is this change backwards compatible or is it a breaking change?
backwards compatible

Special thanks to @j0nas for the help :)
